exercises/luhn/src/example.erl

```diff
@@ -50,24 +51,16 @@ checksum([H | ReversedNumber], even, Total) when H >= $5 ->
 valid(Number) ->
   case is_numeric(Number) of
     true ->
-      case is_min_length(Number) of
-      	true ->
-	      case lists:filter(
-	        fun(C) ->
-	            ($0 =< C) andalso (C =< $9)
-	        end,
-	        Number
-	      ) of
-	        Number2 = [_|_] ->
-	          checksum(Number2) rem 10 == 0;
-	        _ -> false
-	      end;
+      case filter_input(Number) of
+        Number2 = [_,_|_] ->
+          io:format(Number2),
```

Comment:
Please remove this line. I guess you forget to remove it after debugging?

Reply:
That's exactly right!!